Assets/MixedRealityToolkit.Services/InputSimulation/SimulatedHand.cs

```diff
@@ -45,6 +45,8 @@ public void Copy(SimulatedHandData other)
 
         public bool Update(bool isTrackedNew, bool isPinchingNew, HandJointDataGenerator generator)
         {
+            // TODO: DateTime can be quite imprecise, better use Stopwatch.GetTimestamp
```

Comment:
Nit: I believe DateTime has quite high internal resolution, what can be imprecise is the value returned by UtcNow

Reply:
I see, will clarify the comment.